rfc/text/advanced/ap_10_rpc_progressive_calls.md

```diff
@@ -259,3 +259,15 @@ to be matched pairs.
 
 Because they are part of the same call operation, the request ID is the same in all `CALL`,
 `INVOCATION`, `YIELD`, and `ERROR` messages in the above exchange.
+
+
+**Stickiness of Progressive Call Options**
+
+Except for `progress`, items in the `Options` dictionary of the initiating progressive `CALL` shall be effective for
```

Comment:
a positive/concrete list of items in `Options` would be better IMO. a list with the options that are indeed "effective for the call".

Reply:
Well, why not. There are not so many of them. So the reader can see them here in place and doesn't have to jump to other spec sections. The one downside is: if we later add new options - I'm sure everyone will forget to add them here.

Reply:
I'm on the fence for that idea. While it would indeed be convenient to have a list right there, it's going to be a bit of a maintenance problem when new CALL options are added and someone forgets to update that list. Also consider the possibility of the spec being split up into more "modular" RFCs (if that's something you're willing to consider).

We could add a warning that the list may or may not be exhaustive.

If @oberstet insists on that list, then I'll add it. It's no big deal to me either way.